.github/workflows/build.yml

```diff
@@ -0,0 +1,46 @@
+name: Build
+on: [ push ]
+jobs:
+    test:
+        name: Test
+        runs-on: ubuntu-latest
+        steps:
+            -   name: Fetch Sources
+                uses: actions/checkout@v3
+
+            -   name: Gradle Wrapper Validation
+                uses: gradle/wrapper-validation-action@v1
+
+            -   name: Setup Java
+                uses: actions/setup-java@v3
+                with:
+                    distribution: temurin
+                    java-version: 17
+
+            # https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#rate-limits-for-requests-from-github-actions
+            -   name: Show GitHub Actions rate limit
+                shell: bash
+                run: |
+                    curl -s -H "Authorization: Bearer ${{ secrets.GITHUB_TOKEN }}" -i https://api.github.com/users/octocat | grep x-ratelimit
+
+            -   name: Run Linters and Test
+                env:
+                    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+                run: ./gradlew check
+
+            -   name: Verify plugin
+                shell: bash
+                run: ./gradlew verifyPlugin
+
+            -   name: Run Plugin Verifier
+                shell: bash
+                run: ./gradlew runPluginVerifier
```

Comment:
Can you explain for me the difference between `verifyPlugin` and `runPluginVerifier`?

Reply:
`runPluginVerifier`: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-runpluginverifier, it checks binary compatibility with the compatible major versions of the IDEs
`verifyPlugin`: https://plugins.jetbrains.com/docs/intellij/tools-gradle-intellij-plugin.html#tasks-verifyplugin, checks the deployment metadata